tensorboard/backend/application_test.py

```diff
@@ -339,5 +339,30 @@ def testExceptions(self):
       application.TensorBoardWSGIApp(logdir, plugins, multiplexer, 0)
 
 
+class DbTest(tf.test.TestCase):
+
+  def testSqliteDb(self):
+    db_uri = 'sqlite:' + os.path.join(self.get_temp_dir(), 'db')
+    db_module, db_connection_provider = application.get_database_info(db_uri)
+    self.assertTrue(hasattr(db_module, 'Date'))
+    with db_connection_provider() as conn:
+      c = conn.cursor()
+      c.execute('create table peeps (name text)')
+      c.execute('insert into peeps (name) values (?)', ('justine',))
+      conn.commit()
+    with db_connection_provider() as conn:
+      c = conn.cursor()
+      c.execute('select name from peeps')
+      self.assertEquals(('justine',), c.fetchone())
```

Comment:
Maybe we should test that a subsequent call to application.get_database_info(db_uri) can retrieve the same info? ie. the db was not overwritten by the get_database_info call.